src/pudl/output/eia860.py

```diff
@@ -425,6 +425,13 @@ def fill_generator_technology_description(gens_df: pd.DataFrame) -> pd.DataFrame
     ] = (out_df.energy_source_code_1.map(static_fuels))
 
     assert len(out_df) == nrows_orig
+
+    # Assert that at least 95 percent of tech desc rows are filled in
+    if out_df.technology_description.count() \
+        / out_df.technology_description.size \
+            * 100 < 95:
```

Comment:
Why not just use `< 0.95` here? It might also be helpful to put the measured fraction in the AssertionError message. You can also [format floats as percentages](https://datagy.io/python-f-strings/#Formatting_Values_with_Python_f-strings) in f-strings automatically:

```py
x = 0.12321
print(f"This should should show 12.32%: {x:.2%}")
```

Reply:
Easy fix, will do